### ISSUE TITLE
Incorrect display of shared links

### ISSUE THREAD

**Issue by vill** [activeadmin/activeadmin#4995]
Hi there, shared links should looks like this:

![screen shot 2017-06-07 at 20 07 02](https://user-images.githubusercontent.com/2039461/26879959-1c50399a-4bbd-11e7-9b7f-093354f568b5.png)

![screen shot 2017-06-07 at 20 07 14](https://user-images.githubusercontent.com/2039461/26879964-1f99576c-4bbd-11e7-9947-a00144b23728.png)

and so on..

After [this commit](https://github.com/activeadmin/activeadmin/commit/7c14dc39892dedb5560c2a2df828b69ad2b96afd) it began to looks like this:

![screen shot 2017-06-07 at 20 07 35](https://user-images.githubusercontent.com/2039461/26880048-6c8841be-4bbd-11e7-94d0-688bfbf0257a.png)

![screen shot 2017-06-07 at 20 07 47](https://user-images.githubusercontent.com/2039461/26880052-6ead1550-4bbd-11e7-8a5a-ddef828f23cd.png)

and so on..

**Comment by timoschilling** [comment]
@javierjulio can you take a look please.

**Comment by javierjulio** [comment]
That's weird. @vill can you provide more information as versions of AA, devise and Rails? Would be helpful for a sample app containing this. I'm wondering mostly about a possible version issue because I tested this with the login page and others as this demonstrates and it only shows one link per context. Not all of them.

**Comment by vill** [comment]
@javierjulio,

`Rails 5.0.1`
`ActiveAdmin revision: e558e8c08e8c5d625a2d01369bfe2704cecdc296`

The Devise settings in my model require more than one link:

```
class User < ApplicationRecord
  devise :database_authenticatable, :lockable, :timeoutable, :confirmable,
         :recoverable, :rememberable, :trackable, :validatable
end
```
Incorrect display of shared links:

![screen shot 2017-06-07 at 23 15 24](https://user-images.githubusercontent.com/2039461/26889147-59d83474-4bd7-11e7-888f-d40a9e78aa86.png)

Correct display of shared links:

![screen shot 2017-06-07 at 23 16 01](https://user-images.githubusercontent.com/2039461/26889157-648aa0dc-4bd7-11e7-89a6-9a44caee5153.png)

**Comment by javierjulio** [comment]
That makes sense. It seems its based on the devise settings enabled. Thanks @vill.

**Comment by Fivell** [comment]
@javierjulio ,  you gonna fix this?

**Comment by javierjulio** [comment]
@Fivell yes but I'm still having trouble replicating the issue. I'm pretty sure its based on the devise settings enabled. I need to figure out how to get that enabled in the generator for the test app.

**Comment by javierjulio** [comment]
Ok I have this replicated now locally! I'm going to see how I want to tackle this. I might just revert or update the layout.

**Comment by javierjulio** [comment]
Fix in #5057

**Comment by javierjulio** [comment]
Fixing this by bringing back the br tags. Easiest solution for now as I wanted to update the layouts/styles but that's best tackled in #3862/v2.
